### DISCUSSION TITLE
✅ Creating an OSPO Local Meetup - Western Switzerland (French Speaking)

### DISCUSSION THREAD

**Discussion by gravax** [todogroup/ospology#166] [Community Proposals]
Having been contacted by already 2 businesses that feel a need for a structure where to share their experiences and knowledge around their OSPO activities (creation or operation), I think it would make a lot of sense to set up a local meetup for the French speaking part of Switzerland.

It would be run out of Geneva. Would probably start very informally (so far we've been having lunches together)... One of the existing businesses could provide a more formal location (or another organization in Geneva that has shown an interest in joining as well).

Also, once in place, this could radiate a bit to the nearby areas of France... and serve as a stepping stone for setting up similar endeavors in France, and possibly Belgium, and Quebec, and interconnecting them in the future.

**Comment by shillasaebi** [comment]
+1 from me. I think it's a great idea.

**Comment by nicorikken** [comment]
Switzerland already has a strong open source (business) community where you might find interested companies https://www.ch-open.ch/

**Reply by gravax** [reply]
CH/Open is a great team. But most of their events are in German speaking side of Switzerland (Bern, Zurich...)... The idea here is to create something francophone...

**Comment by anajsana** [comment]
Hello @gravax ! Today we hold our Monthly TODO Steering Committee meeting and I'm glad to inform you that your proposal has been approved. The only condition is to create a survey to send to participants following the next meet-up you organize, to gather feedback (@anajsana can help with the questionnaire)

## Next Steps

@anajsana will create a new chapter on the [LF Online Community platform](https://community.linuxfoundation.org/chapters/) for this new OSPO Local Meetup - Western Switzerland in the following two weeks and will give @gravax organizer access.

**Reply by gravax** [reply]
Excellent news! Thanks! Happy to do the survey after the next meetup.

**Reply by anajsana** [reply]
@gravax the Geneva chapter has been created and I've give you organizer permissions to the chapter 👍 https://community.linuxfoundation.org/ospo-local-meetup-western-switzerland-french-speaking/

Please let me know if you have any questions 🙂

**Reply by anajsana** [reply]
Also, a #chapter-geneva channel has been created in [TODO Slack](https://thetodogroup.slack.com/archives/C0448E096AV) for chapter communication.

**Comment by Hleson** [comment]
Thank you!  Gilles, I am here to help.

heather

Heather Leson
***@***.***
Twitter: HeatherLeson <https://twitter.com/HeatherLeson>
Linked In <https://www.linkedin.com/in/heatherleson/>
